Add information about simple requests (#20922)
mdn/content: files/en-us/web/http/cors/index.md

### Simple requests

Some requests don't trigger a {{Glossary("Preflight_request","CORS preflight")}}. Those are called _simple requests_ from the obsolete [CORS spec](https://www.w3.org/TR/2014/REC-cors-20140116/#terminology), though the [Fetch spec](https://fetch.spec.whatwg.org/) (which now defines CORS) doesn't use that term.

> The idea is that the {{HTMLElement("form")}} element from HTML 4.0 (1998) can submit simple requests to any origin, so anyone writing a server must already protect against {{Glossary("CSRF", "cross-site request forgery")}} (CSRF). Then, when cross-site {{domxref("XMLHttpRequest")}} was invented in 2005 followed by {{domxref("fetch")}}, if the request looks like a form submission, then the server doesn't have to opt-in (by responding to a preflight request) to receive the request, since the threat of CSRF is no worse than a form submission. However, the server still must opt-in using {{HTTPHeader("Access-Control-Allow-Origin")}} to _sharing_ the response with the script.

Reviewer: I read this paragraph a few times without understanding it fully. Could you try rewriting it a little bit?

Reviewer: Maybe just a few places that feel unnatural:

```suggestion
The motivation is that the {{HTMLElement("form")}} element from HTML 4.0 (1998) can submit simple requests to any origin, so anyone writing a server must already be protecting against {{Glossary("CSRF", "cross-site request forgery")}} (CSRF). When cross-site {{domxref("XMLHttpRequest")}} was invented in 2005 followed by {{domxref("fetch")}}, the server doesn't have to opt-in (by responding to a preflight request) to receive requests that look like form submissions, since the threat of CSRF of these requests is no worse than that of form submissions. However, the server still must opt-in using {{HTTPHeader("Access-Control-Allow-Origin")}} to _share_ the response with the script.
```

### Simple requests

Some requests don't trigger a {{Glossary("Preflight_request","CORS preflight")}}.
Those are called _simple requests_
from the obsolete [CORS spec](https://www.w3.org/TR/2014/REC-cors-20140116/#terminology),
though the [Fetch spec](https://fetch.spec.whatwg.org/) (which now defines CORS) doesn't use that term.

The idea is that the {{HTMLElement("form")}} element from HTML 4.0 (1998) can submit simple requests to any origin,

> so anyone writing a server must already protect against {{Glossary("CSRF", "cross-site request forgery")}}.

Reviewer 2: ```suggestion
so anyone writing a server must already protect against {{Glossary("CSRF", "cross-site request forgery")}} (CSRF).
```

(since we use the term below)
